bankgreen/urls.py

```diff
@@ -47,7 +47,7 @@
     path("banks/<str:tag>/", views.brand_redirect, name="brand_quicklink"),
     path("sustainable-eco-banks/<str:tag>/", views.brand_redirect, name="brand_quicklink"),
     path("update_success/", views.update_success, name="update_success"),
-    path("rest_api_view/", include("api.urls")),
```

Comment:
This is a clunky name. There's no need to say it's a view. I'd just go ahead and make it or `/api/rest`